Resources/Prototypes/_Goobstation/Changeling/Actions/changeling.yml

```diff
@@ -440,7 +440,7 @@
 - type: entity
   id: ActionBiodegrade
   name: Biodegrade
-  description: Vomit a caustic substance onto any restraints, or someone's face. Costs 30 chemicals.
+  description: We eject a caustic substance, dissolving restraints and spraying those who hold onto us. Costs 30 chemicals.
```

Comment:
i did like the flavor that you were puking, which this loses. could use "spew" or "regurgitate" as the verb instead of eject?

Reply:
changed eject to spew